src/app/App.js

```diff
@@ -3,21 +3,63 @@ import Logo from './components/Logo';
 import createButtonRed from './components/ButtonRed';
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
 import createGameModeName from './components/GameModeName';
+import elemFactory from './utils/elementFactory';
+import render from './utils/render';
 
 const App = ({ options }) => {
   Logo();
 
   VisualImage('../../static/assets/img/modes/people/1.jpg');
 
-  const quiz = document.querySelector('#swquiz-app');
   const buttonPlay = createButtonRed('play the game');
   const buttonRules = createWhiteButtonWithIcon(
     'Rules',
     'fa',
     'fa-graduation-cap',
   );
   const gameModeInfo = createGameModeName('Who is this character?');
-  quiz.append(gameModeInfo, buttonRules, buttonPlay);
+
+  //TODO: to remove after accept
+  /* ------------ Examples of usage: -------------*/
+
+  const firstChildOfTestElement = elemFactory(
+    'div',
+    { id: 'firstChild' },
+    'Pierwsze dziecko',
+  );
+
+  const thirdChildOfTestElement = elemFactory(
+    'ul',
+    {},
+    'Trzecie dziecko - lista elementów:',
+    elemFactory('li', { style: 'color: red' }, 'pierwszy punkt'),
+    elemFactory('li', {}),
+    elemFactory('li', {}, 'trzeci punkt'),
+  );
+
+  const testElement = elemFactory(
+    'div',
+    { class: 'button button--success', id: 'test' },
+    firstChildOfTestElement,
+    'To jest drugie dziecko',
+    thirdChildOfTestElement,
+  );
+
+  const childOfFirstChildOfTestElement = elemFactory(
+    'p',
+    {},
+    'Dziecko pierwszego dziecka',
+  );
+
+  render(
+    'swquiz-app',
+    gameModeInfo,
+    buttonRules,
+    buttonPlay,
+    testElement,
+  );
+
+  render('firstChild', childOfFirstChildOfTestElement);
```

Comment:
So it should be different from parent.append(children) in a way that is accepts text?

Reply:
I see it takes id as an argument, but I am not sure if all elements will have an id? Maybe sometimes its not necessary to add id?

Reply:
Yeah, I assumed that it will be used to render components, and every component should have an id, but maybe it's not the right way.

Reply:
I will dynamically create a table, based on the number of questions. I wasn't going to create an id for every row.. I  understand the need for id, so that if you modify anything later it's easier to 'catch' it.  But I am not sure if that is what we need, I thought append/ append Child would be enough.

Reply:
You are right, it was changed from getElementById to querySelector to be more flexible. So as we were told to avoid using append, with appendChild would be little mess, bacause it accept only one child. And this is where "rendering" comes to the rescue ^^